generators/app/templates/api/server.js

```diff
@@ -0,0 +1,6 @@
+process.env.DEBUG = 'swagger:*';
+var swaggerServer = require('swagger-server');
+var app = swaggerServer('api/DemoSwagger.yaml');
```

Comment:
Please change the naming convention of the file to use dash-convention instead of CamelCase. Some operating systems will dislike this.